components/BtnContainer/BtnContainer.js

```diff
@@ -0,0 +1,20 @@
+import React, {PropTypes} from 'react';
+
+import css from './BtnContainer.css';
+
+export default class BtnContainer {
+  render() {
+    const {children, className, ...props} = this.props;
+
+    return (
+      <button {...props} className={[css.root, className].join(' ')}>
```

Comment:
It's meant for expansion when in use across other applications. Means we can style it as a container rather than be forced into nesting another container:

```jsx
<BtnContainer className="panel">
  <h4>Panel heading</h4>
  <p>Lololol</p>
</BtnContainer>
```

vs

```jsx
<BtnContainer>
  <div className="panel">
    <h4>Panel heading</h4>
    <p>Lololol</p>
  </div>
</BtnContainer>
```

Would seem silly to force the additional container :)